code/__DEFINES/misc.dm

```diff
@@ -141,7 +141,7 @@
 //2 = reduced hud (just hands and intent switcher)
 //3 = no hud (for screenshots)
 
-#define MINERAL_MATERIAL_AMOUNT 2000
+#define MINERAL_MATERIAL_AMOUNT 1000
```

Comment:
NO. FUCK NO.

Do you even understand what the fuck are you doing here? Last time someone did that, it took forever to finally fucking fix every single fucking mat_amount and stop material dupes that were every-fucking-where. And now you are doing this AGAIN.